cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -107,6 +107,38 @@ csrfToken = hmac + "." + randomValue // Add the `randomValue` to the HMAC hash t
 response.setCookie("csrf_token=" + csrfToken + "; Secure") // Set Cookie without HttpOnly flag
 ```
 
+And the following code demonstrates validation of the CSRF token that is sent back from the client:
```

Comment:
This should probably also explicitly refer to "pseudo-code" same as the previous code section.

Reply:
Will update.